code/modules/religion/religion_sects.dm

```diff
@@ -0,0 +1,157 @@
+/datum/religion_sect
+	var/name = "Religious Sect Base Type" // Name of your sect, duh!
+	var/desc = "Oh My! What Do We Have Here?!!?!?!?" // brief description of the sect. Keep it small!
+	var/convert_opener //opening message when someone gets converted
+	var/alignment = ALIGNMENT_GOOD
```

Comment:
holy shit we can port over some nethack and ADOM deities as deity archetypes in a future PR

fuck yeah

Reply:
alignment doesn't do anything atm but I could see for instance a pacifism not being able to use the evil sects